DxDispatch/src/dxdispatch/dxDispatchWrapper.cpp

```diff
@@ -188,6 +189,41 @@ HRESULT DxDispatch::RuntimeClassInitialize(
         m_device->ClearShaderCaches();
     }
 
+    if (m_options->DisableBackgroundProcessing())
+    {
+        HRESULT hr = m_device->D3D()->SetBackgroundProcessingMode(
+            D3D12_BACKGROUND_PROCESSING_MODE_DISABLE_BACKGROUND_WORK,
+            D3D12_MEASUREMENTS_ACTION_KEEP_ALL,
+            nullptr,
+            nullptr
+        );
+
+        if (FAILED(hr))
+        {
+            m_logger->LogError("Failed to disable background processing. Do you have developer mode enabled?");
+            return hr;
+        }
+        else
+        {
+            m_logger->LogInfo("Background processing DISABLED");
+        }
+    }
+
+    if (m_options->SetStablePowerState())
+    {
+        // TODO: smart object to unset state
```

Comment:
Need to fix this before merging the PR

Reply:
Might want to have an else statement to disable this flag if it will persist outside the scope of the process lifetime

Reply:
It shouldn't, but it's a good reminder that the D3D12 device is a singleton and it could be a surprise if some other code in the same process happens to create an ID3D12Device from the same adapter. Moved the cleanup logic to the Device class to make the cleanup more structured.